tests/integration/__fixtures__/site-with-redirect/index.js

```diff
@@ -0,0 +1,25 @@
+const http = require('http')
+
+const server = http.createServer((req, res) => {
+  const pathname = new URL(req.url, 'http://localhost').pathname
+
+  console.log(`Got ${pathname}`)
+
+  if (pathname === '/') {
+    res.write('Root page')
+    res.end()
+  } else if (pathname === '/test/exists') {
+    res.writeHead(302, undefined, { location: '/test/exists/' })
```

Comment:
examples of the site which makes redirect, so then in netlify toml we want to land on this page, which then wants to jump to the next